contracts/Parameters/ParameterManager.sol

```diff
@@ -115,4 +116,11 @@ contract ParameterManager is Initializable, ParameterManagerStorageV2 {
         nativeWrappedToken = _nativeWrappedToken;
         emit NativeWrappedTokenUpdated(_nativeWrappedToken);
     }
+
+    /// @dev Setter for the native wrapped ERC20 token (e.g. WMATIC)
+    function setFreeReactionLimit(uint256 _reactionLimit) external onlyAdmin {
+        require(_reactionLimit >= 0, ZERO_INPUT);
```

Comment:
uint256 will always be >=0... if it can be any value we can prob remove the `require` statement.

Reply:
done